advisories/2.3.0/BRSA-trxsnksws0mo.toml

```diff
@@ -0,0 +1,22 @@
+[advisory]
+id = "BRSA-trxsnksws0mo"
+title = "kernel CVE-2024-41092"
+cve = "CVE-2024-41092"
+severity = "moderate"
+description = "In the Linux kernel, the following vulnerability has been resolved: -  - drm/i915/gt: Fix potential UAF by revoke of fence registers"
```

Comment:
These are coming from the Linux Kernel CNA, can we add a reference to them for folks to follow up if they have more questions? For example, including `https://lore.kernel.org/linux-cve-announce/2024072953-CVE-2024-41092-8bd7@gregkh/` for this description would help get folks one step further to following up if they have questions.